test/extensions/implementation/autograd.py

```diff
@@ -170,6 +180,42 @@ def diag_ggn_mc_batch(self, mc_samples: int) -> List[Tensor]:  # noqa: D102
     def ggn_mc(self, mc_samples: int, chunks: int = 1):  # noqa: D102
         raise NotImplementedError
 
+    @staticmethod
+    def ggn_param_block(outputs: Tensor, loss: Tensor, param: Tensor) -> Tensor:
+        """Calculate the GGN block for a single parameter.
+
+        Args:
+            outputs: Model output.
+            loss: Loss value.
+            param: Parameter of the GGN block.
+
+        Returns:
+            Parameter GGN of shape ``[param.numel(), param.numel()]``.
+        """
+        columns = []
+
+        for i in range(param.numel()):
+            one_hot = zeros_like(param).flatten()
+            one_hot[i] = 1.0
+            one_hot = one_hot.reshape_as(param)
+
+            (column,) = ggn_vector_product_from_plist(loss, outputs, [param], one_hot)
+            columns.append(column.flatten())
+        return reshape(cat(columns), (param.numel(), param.numel()))
```

Comment:
`torch.stack(columns)` should do the job.

Reply:
Fixed in commit
"Some docstring and code cleanup"